cardano-db-sync/src/Cardano/DbSync/Api.hs

```diff
@@ -187,9 +184,6 @@ generateNewEpochEvents env details = do
         , esEpochNo = Strict.Just currentEpochNo
         }
 
-getTrace :: SyncEnv -> Trace IO Text
```

Comment:
yes but unfortunately it won't type check because `envLedger` is a `Maybe` and it seems that `getTrace` isn't solely used in places that don't required and `LedgerEnv` 😞